api/profile/profile.go

```diff
@@ -359,10 +359,15 @@ func profileFromFile(filePath string) (*Profile, error) {
 	if err != nil {
 		return nil, trace.ConvertSystemError(err)
 	}
-	var p *Profile
+	var p Profile
 	if err := yaml.Unmarshal(bytes, &p); err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	if p.Name() == "" {
```

Comment:
Open to suggestions for better ways to detect this.

We could check if `bytes` is empty, if `p` is the zero value, or look at some field other than name.

Reply:
I bet there's plenty of things in the profile yaml that could make the profile unusable if wrong; do we want to catch them all, or is it just an advisory check here?

If it's the latter I think that checking the name should be enough.

Reply:
It is the latter. 👍